app/client/components/transferAccount/StandardTransferAccountList.tsx

```diff
@@ -0,0 +1,329 @@
+    ) {
+      //Set a flag so that if the transferAccounts edit status swaps to success, we'll reload the list
+      this.setState({ awaitingEditSuccess: true });
+    } else if (
+      this.state.awaitingEditSuccess &&
+      this.props.transferAccounts.editStatus.success == true
+    ) {
+      // Update transfer list by re-running server-side search
+      this.setState({ awaitingEditSuccess: false });
+      this.props.loadTransferAccountList({
+        query: {
+          params: this.state.params,
+          search_string: this.state.searchString
+        }
+      });
+    }
+  }
+
+  onSelectChange = (
+    selectedRowKeys: React.Key[],
+    unselectedRowKeys: React.Key[],
+    allSelected: boolean
+  ) => {
+    this.setState({
+      selectedRowKeys,
+      unselectedRowKeys,
+      allSelected
+    });
+  };
+
+  toggleImportModal() {
+    this.setState({ importModalVisible: !this.state.importModalVisible });
+  }
+
+  showBulkTransferModal() {
+    this.setState({ bulkTransferModalVisible: true });
+  }
+
+  handleBulkCancel() {
+    this.setState({ bulkTransferModalVisible: false, amount: 0 });
+  }
+
+  setApproval(approve: boolean) {
+    let { selectedRowKeys, unselectedRowKeys, allSelected } = this.state;
+
+    let include_accounts, exclude_accounts;
+
+    if (allSelected) {
+      //If the "select all" box is true, only specify the accounts to exclude,
+      // as leaving "include_accounts" blank defaults to everything
+      exclude_accounts = unselectedRowKeys;
+    } else {
+      //If the "select all" box is false, only specify the accounts to include.
+      include_accounts = selectedRowKeys;
+    }
+
+    this.props.editTransferAccountRequest({
+      body: {
+        approve,
+        params: this.state.params,
+        search_string: this.state.searchString,
+        include_accounts: include_accounts,
+        exclude_accounts: exclude_accounts
+      },
+      path: "bulk"
+    });
+  }
+
+  updateQueryData(query: Query) {
+    this.setState({
+      params: query.params,
+      searchString: query.searchString
+    });
+  }
+
+  createBulkTransferFromState() {
+    let { selectedRowKeys, unselectedRowKeys, allSelected } = this.state;
+    this.createBulkTransfer(selectedRowKeys, unselectedRowKeys, allSelected);
+  }
+
+  createBulkTransfer(
+    selected: React.Key[],
+    unSelected: React.Key[],
+    allSelected: boolean
+  ) {
+    let include_accounts, exclude_accounts;
+
+    if (allSelected) {
+      //If the "select all" box is true, only specify the accounts to exclude,
+      // as leaving "include_accounts" blank defaults to everything
+      exclude_accounts = unSelected;
+    } else {
+      //If the "select all" box is false, only specify the accounts to include.
+      include_accounts = selected;
+    }
+
+    //We can typecast because the button is only available if the number is set
+    let amount = 100 * (this.state.amount as number);
```

Comment:
Nit: Should this be a utils function? Cents to dollars is pretty common on the FE

Reply:
would need a new utils.ts file because it needs the typing, so i think leave for now....